default/stringtables/en.txt

```diff
@@ -7037,10 +7037,55 @@ RESEARCH_TECH_GUIDE_SHORT_DESC
+
+RESEARCH_TECH_GUIDE_ALLOCATING_RESEARCH_POINTS
+'''<u>Allocating Research Points (RP)</u>
+
+If you double click on an un-researched tech block that tech will be added to the queue. If prerequisite techs are also needed they will be added to the queue as well. You can have multiple different [[metertype METER_RESEARCH]] targets in the queue at once. You can drag techs up and down in the queue to change priorities, remove them from the queue or choose to add techs to the top or bottom of the queue by alt clicking on them and choosing the appropriate option.
+
+[[encyclopedia ENC_TECH]] have two [[metertype METER_RESEARCH]] factors to consider, the total number of RP required to learn that tech and the minimum number of turns that the [[metertype METER_RESEARCH]] for that tech will take. For example, consider a tech which requires a total of 150 RP over at least 5 turns. If your empire is generating 10 RP a turn, it would take 15 turns to research. Instead, if you are generating 100 RP a turn, the same tech will consume 20RP over 5 turns before it is researched. The remaining 80 RP per turn are available for other techs in your queue which are in different tech lines.
+
+If you have unallocated RP, they will be automatically assigned to whatever tech is the cheapest to [[metertype METER_RESEARCH]] and you will be notified when that new tech is available.'''
+
+RESEARCH_TECH_GUIDE_UTILIZING_TECHNOLOGIES
+'''<u>Utilizing Technologies</u>
+
+Completing [[metertype METER_RESEARCH]] on a technology that unlocks an item will only give an empire the knowledge of how to construct that item. Actually constructing such an item may have other requirements, such as access to a certain resource. For example, a [[buildingtype BLD_BLACK_HOLE_POW_GEN]] can only be built in a system surrounding a Black Hole.
+
+If an item has been unlocked and is not available to build, the additional requirements can be checked by displaying the hover text on the items listing in the Production window (with Unavailable selected).
+
+Generally, items unlocked by researched [[encyclopedia ENC_TECH]] do not require [[metertype METER_SUPPLY]] line connections to be available or active (e.g. a building or a planetary meter improvement).'''
+
+RESEARCH_TECH_GUIDE_SHIP_IMPROVEMENTS
+'''<u>Ship Improvements</u>
+
+[[encyclopedia ENC_TECH]] within a technology family, such as Mass Drivers or Lasers, will give automatic improvements to those parts on existing ships. For example, when you [[metertype METER_RESEARCH]] Laser 2, ships with Laser weapons will be automatically upgraded to the new shot damage values, but ships with weapons in other tech families, for example Mass Drivers, will not be changed. Only ships that are in [[metertype METER_SUPPLY]] range will be upgraded, and the upgrade happens on the turn after the [[metertype METER_RESEARCH]] is completed.'''
```

Comment:
meaning of "technology family" unclear, term not used elsewhere; rephrase?

maybe add "additional" before "tech" ?

upgrade happens on the turn after at the earliest, not only on that turn